components/InfoBox/InfoBox.tsx

```diff
@@ -22,7 +22,9 @@ const InfoBox: React.FC<InfoBoxProps> = ({ details, style, className, disabled }
 			{Object.entries(details).map(([key, value]) => (
 				<React.Fragment key={key}>
 					<div>
-						<p className="key">{key}:</p>
+						<div className="key">
+							{key}:{value.tooltip}
```

Comment:
maybe this should be this instead:

```suggestion
{key}{value.tooltip}:
```

Reply:
1) Yeah, let's add some spacing so it looks like this:

![Screen_Shot_2022-05-20_at_09 57 01](https://user-images.githubusercontent.com/548702/169532854-b05f354f-8ee6-4cf1-a95d-b6d1e5b64a77.png)

2) Can we also make sure the tooltip also shows when hovering the (yellow text) dynamic fee value, and not only when hoving the icon?

Reply:
@platschi done - https://github.com/Kwenta/kwenta/pull/858/commits/a1b507aa53256d677e3d46d08a08763f75dbbbf4